lib/av_client/validate_selections.ts

```diff
@@ -48,18 +48,19 @@ export function validateContestSelection( contestConfig: ContestConfig, contestS
 function validateSelectionPile(pile: SelectionPile, markingType: MarkingType, options: OptionContent[]) {
   const isBlank = pile.optionSelections.length === 0
 
+  const getOption = makeGetOption(options)
+
   // Validate blankSubmission
   if( isBlank && markingType.blankSubmission == 'disabled'){
     throw new CorruptSelectionError('Blank submissions are not allowed in this contest')
   }
 
   // Validate that mark count is within bounds
-  if( !isBlank && !withinBounds(markingType.minMarks, pile.optionSelections.length, markingType.maxMarks) ){
+  const bypassMinMarksIfExclusive = getOption(pile.optionSelections[0]).exclusive ? 1 : markingType.minMarks
```

Comment:
couldn't this be called calculatedMinMarks or something, the other name makes it seem like a bool